.github/workflows/main.yml

```diff
@@ -174,6 +174,7 @@ jobs:
           COVERAGE_PROCESS_START: ".coveragerc"
         run: |
           pytest tests/python/ --cov --cov-report=json
+          pytest tests/python/rest_api/test_rq_job_dependencies.py --rebuild --adjusted-server-conf --cov --cov-report=json
```

Comment:
I expect something like

```
ONE_RUNNING_JOB_IN_QUEUE_PER_USER=true pytest tests/python/rest_api/test_rq_job_dependencies.py --rebuild --cov --cov-report=json
```

Need to check. I hope it will work.

Reply:
If we still need to override some settings in docker compose file, I'd rather to leave ONE_RUNNING_JOB_IN_QUEUE_PER_USER there